cmd/server/main.go

```
@@ -4,49 +4,57 @@ import (
-		db, err = sql.Open("postgres", cfg.DatabaseURL)
-		if err != nil {
-			log.Error("failed to open database", "err", err)
-			os.Exit(1)
-		}
-		defer db.Close()
+	db, err := sql.Open("postgres", cfg.DatabaseURL)
+	if err != nil {
+		log.Error("failed to open database", "err", err)
+		os.Exit(1)
+	}
+	defer db.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `db.Close` is not checked

(errcheck)